Palace/MyBooks/MyBooks/MyBooksView.swift

```diff
@@ -19,7 +19,8 @@ struct MyBooksView: View {
   var body: some View {
     NavigationLink(destination: accountScreen, isActive: $model.showAccountScreen) {}
     NavigationLink(destination: searchView, isActive: $model.showSearchSheet) {}
-    
+    NavigationLink(destination: accountPickerList, isActive: $showLibraryAccountView) {}
```

Comment:
I noticed a bug where when the user selected "add account" from the picker view the account list was not being presented